app/assets/stylesheets/mobile/settings.scss

```
@@ -57,3 +57,9 @@
     }
   }
 }
+
+.user-language,
+.user-color-theme,
+.aspect-id {
```

With "replaced by just a class" I meant using one class for all of them together, so you don't need a separate class for each element. Like for example `.settings-dropdown` or something like that. Or as said, it probably also works if you just select all the `select` elements on the settings page, so no extra class is needed at all? That way it would also automatically work if for some reason we would add another dropdown which should also be styled that way on mobile.

```suggestion
.settings-container select {
```

(or better move it with only `select` inside the `.settings-container` block above :) )

---

I now have set the property to all selects inside the settings-container